events/models.py

```diff
@@ -29,6 +29,7 @@ class Event(models.Model):
         default=True,
         help_text="If this is checked all users that attends the event needs to \
         be accepted by an admin.")
+    external_signup_url = models.CharField(max_length=80, blank=True, null=True)
```

Comment:
Now I'm not a Django guru so perhaps there's a good reason for the CharField, otherwise how about an URLField? https://docs.djangoproject.com/ja/1.10/ref/models/fields/#urlfield